src/Drupal/Driver/Cores/Drupal8.php

```diff
@@ -612,4 +612,23 @@ protected function stopCollectingMailSystemMail() {
     }
   }
 
+  /**
+   * Store the original value for a piece of configuration.
+   *
+   * @param string $name
+   *   The name of the configuration.
+   * @param mixed $value
+   *   The original value of the configuration.
+   *
+   * @return bool
+   *   Whether or not $value is now stored as the original configuration value.
+   */
+  protected function storeOriginalConfiguration($name, $value) {
+    if (!isset($this->originalConfiguration[$name])) {
+      $this->originalConfiguration[$name] = $value;
+      return TRUE;
+    }
+    return ($this->originalConfiguration[$name] === $value);
```

Comment:
Is there a reason to have a return value at all here? It doesn't appear to be used in this change...